lib/database-ops.js

```diff
@@ -138,10 +138,16 @@ var startDatabaseClient = function(callback, options) {
             else {
                 imageEntryCollection = database.collection("image-entries");
                 imageEntryCollection.dropIndex("expire_index").then(() => {
-                    createIndexForImageEntriesCollection(imageEntryCollection)
+                    createIndexForImageEntriesCollection(imageEntryCollection);
                     finishInitialization();
                 }).catch((err) => {
-                    callback(err);
+                    if(err.codeName === "IndexNotFound") {
```

Comment:
Would be nice to have a log statement here so we know if index is being created when there wasn't one previously.